markdown_lint/linter.py

```
@@ -0,0 +1,317 @@
+"""Core markdown linter implementation."""
+
+import re
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional, Pattern, Set, Tuple, Union
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import

(F401)